src/components/App.js

```diff
@@ -0,0 +1,17 @@
+/* eslint linebreak-style: ["error", "windows"] */
+
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <div className="App-header">
+      <h1>CALCULATOR</h1>
+      <Display />
+      <ButtonPanel />
+    </div>
```

Comment:
- [ ] [REQUIRED] These children should be wrapped inside React fragments
![image](https://user-images.githubusercontent.com/73254040/99707572-418f3d00-2a62-11eb-84e6-1ef0bba65f06.png)
[Source](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/44896092/)